sources/packages/backend/apps/api/src/app.module.ts

```diff
@@ -60,12 +65,18 @@ import { QueueModule } from "@sims/services/queue";
       },
     ]),
   ],
-  controllers: [AppController, ConfigController, DynamicFormController],
+  controllers: [
+    AppController,
+    ConfigController,
+    DynamicFormController,
+    AnnouncementsController,
+  ],
   providers: [
     AppService,
     UserService,
     BCeIDServiceProvider,
     FormService,
+    AnnouncementsService,
```

Comment:
I have a doubt here. Doesn't adding the service in this module resolve it for the other modules (aest / institution / student)?

Reply:
Adding service here doesn't resolve the dependency in other modules, but the `AnnouncementsService` is not required to be added in other modules as it is required by the controller which is added only to app module.

Reply:
Thanks @dheepak-aot . Got it. 👍